pkgs/by-name/so/sockstat/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sockstat";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "mezantrop";
+    repo = "sockstat";
+    tag = finalAttrs.version;
+    sha256 = "sha256-7VfideKNWlWb9nnAL2TK7HiD0T5EJsQiagT2kPMwrdA=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm0755 sockstat -t $out/bin
```

```suggestion
    installBin sockstat
```

And add `installShellFiles` to `nativeBuildInputs`.